client/v2/algod/algod.go

```diff
@@ -80,8 +80,8 @@ func (c *Client) Block(round uint64) *Block {
 	return &Block{c: c, round: round}
 }
 
-func (c *Client) GetProof(round uint64, txid string) *GetProof {
-	return &GetProof{c: c, round: round, txid: txid}
+func (c *Client) GetTransactionProof(round uint64, txid string) *GetTransactionProof {
```

Comment:
I strongly suggest that we should consider adding the hash type as a parameter even though it is an optional parameter to the REST.
Bridge operators would most use the SHA256 proofs and not the default one.